packages/xsnap/build.env

```diff
@@ -1,4 +1,4 @@
 MODDABLE_URL=https://github.com/agoric-labs/moddable.git
 MODDABLE_COMMIT_HASH=f6c5951fc055e4ca592b9166b9ae3cbb9cca6bf0
 XSNAP_NATIVE_URL=https://github.com/agoric-labs/xsnap-pub
-XSNAP_NATIVE_COMMIT_HASH=2d8ccb76b8508e490d9e03972bb4c64f402d5135
+XSNAP_NATIVE_COMMIT_HASH=b0484e9c58b197be13eabb4dd4bc92188fe0d153
```

Comment:
Please update to the [head of the `Agoric` branch](https://github.com/agoric-labs/xsnap-pub/commits/Agoric), aka `87566073e377cbc6fe3f874e6cbed990de3f07b6`.

Reply:
Aye, now aligned with the commit hash after the rebase+and+merge in xsnap-pub.